pkg/plugin/cmd.go

```diff
@@ -196,13 +235,40 @@ func (o *DebugOptions) Validate() error {
 	return nil
 }
 
+// Run run
 func (o *DebugOptions) Run() error {
-
 	pod, err := o.CoreClient.Pods(o.Namespace).Get(o.PodName, v1.GetOptions{})
 	if err != nil {
 		return err
 	}
+	agents, err := o.CoreClient.Pods(v1.NamespaceAll).List(v1.ListOptions{
+		LabelSelector: fmt.Sprintf("app=%s", o.AppName),
+	})
+	if err != nil {
+		return err
+	}
 
+	// get the agent pod which is in the same node with specify pod
+	var agent *corev1.Pod
+	for _, a := range agents.Items {
+		if a.Spec.NodeName == pod.Spec.NodeName {
+			agent = &a
+			break
+		}
+	}
+	if agent == nil {
+		return fmt.Errorf("there is no agent pod in the same node with your speficy pod %s", o.PodName)
+	}
+	fmt.Printf("pod %s PodIP %s, agentPodIP %s\n", o.PodName, pod.Status.PodIP, agent.Status.HostIP)
+	err = o.runPortForward(agent)
+	if err != nil {
+		return err
+	}
+	// client can't access the node ip in the k8s cluster sometimes,
+	// than we use forward ports to connect the specified pod and that will listen
+	// on specified ports in localhost, the ports can not access until receive the
+	// ready signal
+	<-o.ReadyChannel
```

Comment:
aha, maybe I haven't explain it clearly, by saying "log", I mean we could print some message in the command line to tell user he/she may have to wait for the port-forward complete, in this way, if port-forward takes too long, the user can figure out what is going on.

Reply:
I will commit a log line because that change is trivial😁

Reply:
emmm, i'll be careless 😁